src/screens/Home/components/HomeMenuItem.tsx

```diff
@@ -38,54 +44,67 @@ const styles = (theme: Theme) =>
     noAlert: {
       opacity: '50%',
     },
-    sublabel: {
-      color: '#BDBDBD',
-    },
-    icon: {
-      width: '20px',
-      height: '20px',
+    badgeStyles: {
+      height: 24,
+      width: 24,
       color: theme.palette.common.white,
       display: 'flex',
       alignItems: 'center',
       justifyContent: 'center',
       borderRadius: '50%',
-      backgroundColor: '#FF6142',
+      backgroundColor: theme.palette.error.main,
+    },
+    checked: {
+      backgroundColor: theme.palette.primary.main,
     },
-    grey: {
-      backgroundColor: '#BDBDBD',
+    innerContent: {
+      display: 'flex',
+      justifyContent: 'flex-start',
+      alignItems: 'center',
     },
   });
 
 function HomeMenuItem(props: HomeMenuItemProps) {
-  const { label, sublabels, noBadge, classes } = props;
-  const amount = props.amount
-    ? sublabels
-      ? sublabels.reduce(function (amt: number, curr: sublabel) {
-          return amt + curr.amount;
-        }, 0)
-      : props.amount
-    : null;
+  const { label, noBadge, classes, iconType, amount } = props;
+  const renderIcons = () => {
+    if (iconType === 'meter' || iconType === 'collect') {
+      return (
+        <AccountBoxIcon fontSize='large' color="primary" />
+      );
+    }
+    if (iconType === 'maintenance') {
+      return (
+        <BuildIcon fontSize='large' color="primary" />
+      );
+    }
+    if (iconType === 'incident') {
+      return (
+        <WarningIcon fontSize='large' color="primary" />
+      );
+    }
+  }
+
+  const getRightBadge = () => {
+    if (amount === 0) {
+      return <CheckCircleIcon color="primary" />
+    }
+
+    // We cap at 99 so the circle spacing doesn't break
+    return <div className={classes.badgeStyles}>
+      <Typography variant="body2">{amount > 99 ? '99+' : amount}</Typography>
+    </div>
+  }
 
   return (
     <ButtonBase className={classes.root}>
-      <div className={`${classes.content} ${noBadge ? null : amount ? null : classes.noAlert}`}>
-        <div>
-          <Typography variant="h2">{label}</Typography>
-          {sublabels
-            ? sublabels.map((sl: sublabel, index: number) => (
-                <Typography className={classes.sublabel} variant="body1" key={index}>
-                  {sl.amount} {sl.label}
-                </Typography>
-              ))
-            : null}
+      <div className={classes.content}>
```

Comment:
This could be converted to use the `Card` component + `CardContent` and `CardActions` etc. which would take care of a lot of the custom styles that are used here and keep it consistent with the other similar cards throughout the app. This component isn't very bloated so I'm okay with leaving it like this, but just a thought.

Reply:
I put this as a comment to consider in case the component gets expanded